src/config.rs

```diff
@@ -172,7 +174,7 @@ impl Default for DbOptions {
             compactor_options: Some(CompactorOptions::default()),
             compression_codec: None,
             object_store_cache_options: ObjectStoreCacheOptions::default(),
-            block_cache_options: Some(DbCacheOptions::default()),
+            block_cache: Some(Arc::new(MokaCache::new())),
```

Comment:
WDYT about using a `#[cfg(feature = "moka")]` here, and have a similar block for `#[cfg(feature = "foyer")]` (with Foyer default), and default to `None` if neither is set?

Same for the `use crate::db_cache::moka::MokaCache;` above. That will fail if we don't have the `moka` feature set.

Reply:
I've changed this default. It's actually a little more tricky because you have to decide what the default is when both features are enabled, and when no feature is enabled. Hopefully, it looks good now.